pkg/operator/ceph/csi/controller.go

```diff
@@ -299,6 +310,23 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 		if err != nil {
 			return opcontroller.ImmediateRetryResult, errors.Wrapf(err, "failed to update CSI driver options for cluster %q", cluster.Name)
 		}
+
+		// Skip the new CSI-operator creation when holder pod is enabled until multus support is added in the CSI operator
+		if EnableCSIOperator() {
+			logger.Info("disabling csi-driver as `EnableCSIOperator` is `true`")
+			EnableCephFS = false
```

Comment:
Don't we still need these params to decided which `Driver` resource to be created for the csi-operator? @subhamkrai

Reply:
What I meant was that I was planning to use these fields to create the corresponding `Drivers` for the csi operator. For example, only create rbd driver resource if `EnableRBD` is `true`.

Reply:
Me and Santosh already discussed this, so closing this comment. Thanks